sql/src/main/java/io/crate/analyze/RefreshTableAnalyzedStatement.java

```diff
@@ -21,18 +21,22 @@
 
 package io.crate.analyze;
 
-import java.util.Set;
+import io.crate.expression.symbol.Symbol;
+import io.crate.metadata.doc.DocTableInfo;
+import io.crate.sql.tree.Table;
+
+import java.util.Map;
 
 public class RefreshTableAnalyzedStatement implements DDLStatement {
```

Comment:
maybe we could rename this to `AnalyzedRefreshTable` to be aligned with the `AnalyzedCreateTable`?

Reply:
Isn't it required to override the `isUnboundPlanningSupported` interface method?